packages/fluentui/react-northstar/src/components/Form/utils/formFieldBase.tsx

```diff
@@ -99,15 +104,22 @@ const _FormFieldBase = compose<'div', FormFieldBaseProps, {}, {}, {}>(
               ...slotProps.label,
             }),
         })}
-        {createShorthand(composeOptions.slots.control, control || {}, {
-          defaultProps: () =>
-            getA11yProps('control', {
-              error: !!errorMessage || null,
-              ref,
-              ...unhandledProps,
-              ...slotProps.control,
-            }),
-        })}
+        {/**
+         * When there's a message for the input the labelId and messageId should be consistent in the
+         * aria-labelledby attribute (aria-labelledby="labelID messageID") therefore we need to pass it down
+         * for components like input that are generating its own label internally
+         */}
+        <FormFieldBaseProvider value={childProps}>
```

Comment:
`childProps` is an unstable object and will cause the rerender of `composeOptions.slots.control` every time

Reply:
Fixed